src/constants.ts

```diff
@@ -17,9 +17,7 @@ export const CONFIG_INTRO = `===============================
 Nightwatch Configuration Wizard
 ===============================
 
-Just answer a few questions to get started with Nightwatch:
-
-We'll setup everything for you :-)
+Initializing a new project in %s...
```

Comment:
The config wizard does not necessarily initialize a new project. It can be used in existing Nightwatch projects as well, to add some new capabilities for example like adding mobile support or support to test on BrowserStack.